scripts/api.js

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const fs         = require('fs');
+const path       = require('path');
+const yaml       = require('js-yaml');
+const config     = require('../config');
+const sri        = require('./sri.js');
+
+const configFile = config.getConfigPath('_api.yml');
+
+const semver = require('semver');
+
+function getDirectoryNames(source, ignore = []) {
+    return fs.readdirSync(source).map((name) => ignore.indexOf(name) === -1 && name).filter((name) => {
```

Comment:
Can you indent a bit more your code here, for example:
```js
return fs.readdirSync(source)
  .map((name) => ignore.indexOf(name) === -1 && name)
  .filter((name) => {});
```

better for the readability